### PR TITLE
Edits: Update index.md

### PR DESCRIPTION
**Pull request by Kellylorenebaker**
@Thraka

This looks great! I tried to tighten up the intro a bit, but otherwise, I just made basic copy edit suggestions. Not sure how closely your content model aligns to the all-up Azure model, but the Azure model usually includes a Next steps section at the end of each article. If this isn't relevant to your docset, I wouldn't worry about it. Let me know if you have any questions. Thanks!

Per work item: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1608924

### CODE REVIEW

**Review by Thraka — APPROVED** [review]
@Kellylorenebaker Looks great, thank you! A lot of this content was written years ago and hasn't been modernized. I appreciate your time here!

**Comment by Thraka** [comment]
Contributes to #14676

File: docs/desktop-wpf/overview/index.md [hunk 1]
```diff
@@ -134,6 +134,6 @@ For more information, see [Styles and templates](../fundamentals/styles-template
 
 WPF resources are objects that can be reused in different places in your application. Examples of resources include styles, templates, and color brushes. Resources can be both defined and referenced in code and in XAML format.
 
-Every framework-level element (<xref:System.Windows.FrameworkElement> or <xref:System.Windows.FrameworkContentElement>) has a `Resources` property (which is a <xref:System.Windows.ResourceDictionary> type) that contains defined resources. Since all elements inherit from a framework-level element, all elements can define resources. It's common though to define resources on a root element of a XAML document.
+Every framework-level element (<xref:System.Windows.FrameworkElement> or <xref:System.Windows.FrameworkContentElement>) has a `Resources` property (which is a <xref:System.Windows.ResourceDictionary> type) that contains defined resources. Since all elements inherit from a framework-level element, all elements can define resources. It's most common, however, to define resources on a root element of a XAML document.
 
 <!--For more information, see [XAML Resources](../../../framework/wpf/advanced/xaml-resources.md).-->
```
Comment (Thraka) [inline docs/desktop-wpf/overview/index.md]:
```suggestion
<!--For more information, see [XAML Resources](../../../framework/wpf/advanced/xaml-resources.md).-->

## Next steps

- [Create a WPF application](https://docs.microsoft.com/visualstudio/get-started/csharp/tutorial-wpf?toc=/dotnet/desktop-wpf/toc.json&bc=/dotnet/breadcrumb/toc.json)
- [Explore the differences from .NET Framework](migration/differences-from-net-framework.md)
- [Learn about XAML](fundamentals/xaml.md)
```

Comment (Thraka) [reply docs/desktop-wpf/overview/index.md]: Thank you for the Next Steps reminder. I've added that.

**Review by Kellylorenebaker — COMMENTED** [review]
Looks great!

**Pull request merged**